app/uk/gov/hmrc/servicedependencies/ServiceDependenciesController.scala

```diff
@@ -47,22 +49,26 @@ object RepositoryDependencies {
   implicit val format = Json.format[RepositoryDependencies]
 }
 
-trait ServiceDependenciesController extends BaseController {
 
-  lazy val logger = LoggerFactory.getLogger(this.getClass)
+@Singleton
+class ServiceDependenciesController @Inject()(configuration: Configuration,
+                                              dependencyDataUpdatingService: DependencyDataUpdatingService,
+                                              config: ServiceDependenciesConfig) extends BaseController {
 
-  private val doneResult = Ok("Done")
+//  protected val config = new ServiceDependenciesConfig("/dependency-versions-config.json", configuration)
 
-  def dependencyDataUpdatingService: DependencyDataUpdatingService
+//  lazy val dependencyDataUpdatingService: DependencyDataUpdatingService = new DependencyDataUpdatingService(config, mongo.mongoConnector.db)
 
-	implicit val environmentDependencyWrites = Json.writes[EnvironmentDependency]
-	implicit val serviceDependenciesWrites = Json.writes[ServiceDependencies]
+  val logger = LoggerFactory.getLogger(this.getClass)
```

Comment:
Do you need this? can you use play.api.Logger object instead?

Reply:
Does it really matter?